packages/mock-addon-docs/package.json

```diff
@@ -15,20 +15,21 @@
   "author": "Farhad Yasir(nutboltu)",
   "license": "MIT",
   "devDependencies": {
-    "@storybook/addon-actions": "^7.4.2",
-    "@storybook/addon-essentials": "^7.4.2",
-    "@storybook/addon-interactions": "^7.4.2",
-    "@storybook/addon-links": "^7.4.2",
-    "@storybook/react-webpack5": "^7.4.2",
+    "@storybook/addon-actions": "^8.0.5",
```

Comment:
I think it's best to keep versions of the `@storybook` packages packages aligned, so `"^8.0.8"` for all of them, including the main `storybook` package.

Reply:
Agree, done.